### ISSUE TITLE
Remove versioned resources after maven build system

### ISSUE THREAD

**Issue by javierfernandes** [uqbar-project/wollok#57]
.project (?) .classpath (?), .settings (?)

**Comment by javierfernandes** [comment]
Here a command to see which files are currently versioned

```bash
jfernandes@Macinaural:~/dev/data/repo/wollok-dev$ for i in `find . -name ".classpath"`; do echo $i; git log --oneline $i | head -n 1; done
./org.eclipse.xtend/.classpath
./org.eclipse.xtend.core/.classpath
./org.eclipse.xtend.ide/.classpath
./org.uqbar.project.wollok/.classpath
74a774b Merge branch 'wollok-game' of https://github.com/ProyectoFinal2015/wollok into ProyectoFinal2015-wollok-game
./org.uqbar.project.wollok.game/.classpath
./org.uqbar.project.wollok.game.ui/.classpath
./org.uqbar.project.wollok.launch/.classpath
3e69293 Fixing dependencies
./org.uqbar.project.wollok.lib/.classpath
74a774b Merge branch 'wollok-game' of https://github.com/ProyectoFinal2015/wollok into ProyectoFinal2015-wollok-game
./org.uqbar.project.wollok.product/.classpath
74a774b Merge branch 'wollok-game' of https://github.com/ProyectoFinal2015/wollok into ProyectoFinal2015-wollok-game
./org.uqbar.project.wollok.product.ui/.classpath
582913e Fixes #476 filtering unwanted items from wizards. I had to create a new feature so that this logic will only be part of the 'wollok as a product' and not wollok as plugins. We don't want to remove thing from a custom eclipse in where the user has added wollok. I've also found some build issues with the new version of tycho and the way wollok game was handling dependencies. So I falled back to a more manual solution
./org.uqbar.project.wollok.tests/.classpath
74a774b Merge branch 'wollok-game' of https://github.com/ProyectoFinal2015/wollok into ProyectoFinal2015-wollok-game
./org.uqbar.project.wollok.typeSystem/.classpath
./org.uqbar.project.wollok.typeSystem.xsemantics/.classpath
74a774b Merge branch 'wollok-game' of https://github.com/ProyectoFinal2015/wollok into ProyectoFinal2015-wollok-game
./org.uqbar.project.wollok.typeSystem.xsemantics.ui/.classpath
c11a053 Ignored jacoco from eclipse maven build
./org.uqbar.project.wollok.ui/.classpath
74a774b Merge branch 'wollok-game' of https://github.com/ProyectoFinal2015/wollok into ProyectoFinal2015-wollok-game
./org.uqbar.project.wollok.ui.diagrams/.classpath
74a774b Merge branch 'wollok-game' of https://github.com/ProyectoFinal2015/wollok into ProyectoFinal2015-wollok-game
./org.uqbar.project.wollok.ui.launch/.classpath
3e69293 Fixing dependencies
./org.uqbar.project.xinterpreter/.classpath
4ad1e48 Fixes #499 splitting xinterpreter
./wollok-tests/.classpath
3e69293 Fixing dependencies```

**Comment by javierfernandes** [comment]
Command to remove them from git

```
for i in `find . -name ".classpath"`; do echo $i; git rm --cached $i ; done
```
